checkov/common/parsers/multiline_parser.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+from abc import ABC
+from abc import abstractmethod
+
+if TYPE_CHECKING:
+    from detect_secrets.util.code_snippet import CodeSnippet
+
+
+class BaseMultiLineParser(ABC):
+    @abstractmethod
+    def __init__(self) -> None:
+        pass
```

Comment:
no need to define this, if a subclass needs an `__init__` they are free to use it.